synapse/events/builder.py

```diff
@@ -128,6 +128,7 @@ async def build(
                 state_filter=StateFilter.from_types(
                     auth_types_for_event(self.room_version, self)
                 ),
+                await_full_state=False,
```

Comment:
The only other call of `compute_state_after_events` is in `RoomMemberHandler.update_membership_locked`.
Membership being a lot more tricky, I guess we don't want to use `await_full_state=False` in this case so we can't just hardcode that directly in `compute_state_after_events`.